Cloudtrail/main.tf

```diff
@@ -0,0 +1,86 @@
+terraform {
+  backend "s3" {
+    bucket = "cloud-platform-cloudtrail-tf-state"
+    region = "eu-west-1"
+    key    = "terraform.tfstate"
+  }
+}
+
+provider "aws" {
+    region = "eu-west-1"
+}
+
+data "aws_caller_identity" "current" {}
+
+resource "aws_cloudtrail" "cloud-platform-cloudtrail" {
+  name                          = "cp-cloudtrail"
+  s3_bucket_name                = "${aws_s3_bucket.cloudtrail-bucket.id}"
+  include_global_service_events = true
+  is_multi_region_trail         = true
+  enable_log_file_validation    = true
+  tags {
+        business-unit           = "${var.business-unit}"
+        owner                   = "${var.team_name}"
+        infrastructure-support  = "${var.infrastructure-support}"
+    }
+}
+resource "aws_s3_bucket" "cloudtrail-bucket" {
+  bucket        = "cloud-platform-cloudtrail-moj"
+  force_destroy = false
+  tags {
+        business-unit           = "${var.business-unit}"
+        owner                   = "${var.team_name}"
+        infrastructure-support  = "${var.infrastructure-support}"
+    }
+  lifecycle_rule {
+    id                                     = "logs-transition"
+    prefix                                 = ""
+    abort_incomplete_multipart_upload_days = 7
+    enabled                                = true
+
+    transition {
+      days          = 30
+      storage_class = "STANDARD_IA"
+    }
+
+    transition {
```

Comment:
Let's not go into glacier. It's then painful to retrieve anything. I doubt there's any real cost savings.

Reply:
After a chat, this has been agreed and pushed into cb8016a.